lib/mocha.js

```diff
@@ -12,6 +12,8 @@ var escapeRe = require('escape-string-regexp');
 var path = require('path');
 var reporters = require('./reporters');
 var utils = require('./utils');
+var Random = require('random-js');
+var MAX_SAFE_INTEGER = 0x1fffffffffffff;
 
```

Comment:
From https://www.npmjs.com/package/random-js#distributions

> - `Random.integer(min, max)(engine)`: Produce an integer within the inclusive range [`min`, `max`]. `min` can be at its minimum -9007199254740992 (-2 ** 53). `max` can be at its maximum 9007199254740992 (2 ** 53).

`Number.MAX_SAFE_INTEGER` is the same value.